docs/CARS.md

```diff
@@ -141,7 +141,7 @@
 | SEAT      | Leon 2014-2020                | Driver Assistance | Stock            | 0mph               | 0mph         |
 | Škoda     | Kamiq 2021<sup>2</sup>        | Driver Assistance | Stock            | 0mph               | 0mph         |
 | Škoda     | Karoq 2019                    | Driver Assistance | Stock            | 0mph               | 0mph         |
-| Škoda     | Kodiaq 2018-19                | Driver Assistance | Stock            | 0mph               | 0mph         |
+| Škoda     | Kodiaq 2018-19, 2021          | Driver Assistance | Stock            | 0mph               | 0mph         |
```

Comment:
comma allows us to fill single-year support gaps as long as we're confident the missing year will work, and I am.

```suggestion
| Škoda     | Kodiaq 2018-21                | Driver Assistance | Stock            | 0mph               | 0mph         |
```